app/views/candidates/show.html.erb

```diff
@@ -0,0 +1,59 @@
+<section class="vh-100" style="background-color: #f4f5f7;">
+  <div class="container py-5 h-100">
+    <div class="row d-flex justify-content-center align-items-center h-100">
+      <div class="col col-lg-6 mb-4 mb-lg-0">
+        <div class="card mb-3" style="border-radius: .5rem;">
+          <div class="row g-0">
+            <div class="col-md-4 gradient-custom text-center text-white"
+              style="border-top-left-radius: .5rem; border-bottom-left-radius: .5rem;">
+              <img src="https://mdbcdn.b-cdn.net/img/Photos/new-templates/bootstrap-chat/ava1-bg.webp"
+                alt="Avatar" class="img-fluid my-5" style="width: 80px;" />
+            </div>
+            <div class="col-md-8">
+              <div class="card-body p-4">
+                <h4>Information</h4>
+                <hr class="mt-0 mb-4">
+                <div class="row pt-1">
+                  <div>
+                    <h6>Name</h6>
+                    <p class="text-muted"><%= @candidate.name %></p>
+                  </div>
+                  <div>
+                    <h6>Email</h6>
+                    <p class="text-muted"><%= @candidate.email %></p>
+                  </div>
+                  <div>
+                    <h6>Contact</h6>
+                    <p class="text-muted"><%= @candidate.contact_number %></p>
+                  </div>
+                  <div>
+                    <h6>Address</h6>
+                    <p class="text-muted"><%= @candidate.address %></p>
+                  </div>
+                  <div>
+                    <h6>Experience in year(s)</h6>
+                    <p class="text-muted"><%= @candidate.experience %></p>
+                  </div>
+                  <div>
+                    <button type="button" class="btn btn-primary">
+                      <%= link_to 'Edit Candidate Details', edit_candidate_path, class: "nav-link active" %>
```

Comment:
question: how this link is working without passing @candidate in it? it should be `edit_candidate_path(@candidate)`?, same question for line no 45 and 49.